ui/StatusQ/src/stringutilsinternal.cpp

```diff
@@ -1,7 +1,13 @@
 #include "StatusQ/stringutilsinternal.h"
 
-StringUtilsInternal::StringUtilsInternal(QObject* parent)
-    : QObject(parent)
+#include <QFile>
+#include <QFileSelector>
+#include <QQmlEngine>
+#include <QQmlFileSelector>
+
+StringUtilsInternal::StringUtilsInternal(QQmlEngine *engine, QObject* parent)
```

Comment:
```suggestion
StringUtilsInternal::StringUtilsInternal(QQmlEngine *engine, QObject *parent)
```

Reply:
Hmm, I don't understand. If clang format works automatically for you, then why in a single row there are two opposite approaches used?

Reply:
Right... strange :) Will fixup

Reply:
Fixed